app/src/main/java/salesian/university/broker/TopicManager.java

```diff
@@ -0,0 +1,27 @@
+package salesian.university.broker;
+
+import java.util.*;
```

Comment:
It'd be good if you just import those objects you use into the class and not import all.